src/openforms/registrations/contrib/json_dump/plugin.py

```diff
@@ -65,7 +65,8 @@ def register_submission(
             if key in options["variables"]
         }
         values_schema = generate_json_schema(submission.form, options["variables"])
-        post_process(values, values_schema, submission)
+        transform_to_list = options.get("transform_to_list")
```

Comment:
```suggestion
        transform_to_list = options["transform_to_list"]
```
with the change to types above :)